src/main/java/org/jenkinsci/test/acceptance/po/PluginManager.java

```diff
@@ -247,16 +252,16 @@ public boolean installPlugins(final PluginSpec... specs) throws UnableToResolveD
 
                 // Temporary until https://github.com/jenkinsci/jenkins/pull/8025 is in LTS
                 if (find(by.button("Download now and install after restart")) != null) {
-                    clickButton("Download now and install after restart");
+                    control(by.button("Download now and install after restart")).clickAndWaitToBecomeStale();
                 } else {
-                    clickButton("Update");
+                    control(by.button("Update")).clickAndWaitToBecomeStale();
                 }
             }
         }
 
         // Jenkins will be restarted if necessary
         boolean hasBeenRestarted = new UpdateCenter(jenkins).waitForInstallationToComplete(specs);
-        if (!hasBeenRestarted && specs.length > 0 && jenkins.getVersion().isNewerThan(new VersionNumber("2.188"))) {
```

Comment:
supported Jenkins for the ath are passed this version now, so removed this condition whilst I was here.